Triangulation_on_sphere_2/include/CGAL/Triangulation_on_sphere_2.h

```diff
@@ -964,18 +964,12 @@ march_locate_2D(Face_handle f,
         li = (o0 == COLLINEAR) ? 2 : (o1 == COLLINEAR) ? 0 : 1;
         break;
       }
-      case 2:
+      default:   // i.e., case 2:
       {
         lt = VERTEX;
         li = (o0 != COLLINEAR) ? 2 : (o1 != COLLINEAR) ? 0 : 1;
         break;
       }
-      default:
-      {
-        // impossible
-        CGAL_assertion(false);
-        return f;
-      }
     }
```

Comment:
For the compiler, those two versions of the code are different.

* With
  
  ```c++
  switch(sum)
  {
    case 0: // ...
    case 1: // ...
    case 2: // ...
    default: CGAL_unreachable();
  }
  ```
  
  then the compiler knows `sum` can only be 0, 1, or 2.
* With
  
  ```c++
  switch(sum)
  {
    case 0: // ...
    case 1: // ...
    case 2: // ... // do not forget the break!
    default: CGAL_unreachable();
  }
  ```
  
  then the compiler has no knowledge about values of `sum`, and might produce a slightly less efficient code (and maybe warnings).

In that case, where `sum` is initialized with:

```c++
    int sum =  ((o0 == COLLINEAR) ? 1 : 0)
             + ((o1 == COLLINEAR) ? 1 : 0)
+ ((o2 == COLLINEAR) ? 1 : 0);
```

maybe the compiler was already able to deduce the value range of `sum`. The version with `default: CGAL_unreachable();` is more explicit.

Reply:
I reverted this change.